compressor/tests/test_offline.py

```diff
@@ -655,3 +656,31 @@ def _test_offline(self, engine):
         count, result = CompressCommand().compress(
             log=self.log, verbosity=self.verbosity, engine=engine)
         self.assertEqual(count, 1)
+
+
+@skipIf(not _TEST_JINJA2, "Test only run if we are testing Jinja2")
+class TestCompressCommand(OfflineTestCaseMixin, TestCase):
+    templates_dir = "test_compress_command"
+
+    def _test_offline(self, engine):
+        raise SkipTest("Not utilized for this test case")
+
+    def test_multiple_engines(self):
+        opts = {
+            "force": True,
+            "verbosity": 0,
+            "log": StringIO(),
+        }
+
+        call_command('compress', engines=["django"], **opts)
+        manifest_django = get_offline_manifest()
+
+        call_command('compress', engines=["jinja2"], **opts)
+        manifest_jinja2 = get_offline_manifest()
+
+        call_command('compress', engines=["django", "jinja2"], **opts)
+        manifest_both = get_offline_manifest()
+
+        self.assertEqual(manifest_django, {'8464063aa0729700fca0452e009582af': '<script type="text/javascript" src="/static/CACHE/js/662b9ce354e4.js"></script>'})
```

Comment:
Please wrap that at 79 columns (we still have files not respecting that in the project, but I tried to keep this one as clean as possible)